lib/exercise_test_cases.rb

```diff
@@ -15,6 +16,6 @@ def parsed_json_cases
   end
 
   def to_a
-    parsed_json_cases.map { |test_case| case_classname.new(test_case) }
+    parsed_json_cases.map.with_index { |test_case, index| case_classname.new(test_case, index) }
```

Comment:
Yeah I was kind of torn about whether or not to keep the skip method - I think there is benefit to both ways for sure.